### ISSUE TITLE
Valid folder not found in the given dataset for PA_02

### ISSUE THREAD

**Issue by MahmodulRatul** [debajyotikarmaker/cvpr_4234#14]
In the dataset of following link, I did not found the valid folder. Only train and test is there.
https://www.kaggle.com/grassknoted/asl-alphabet
Anyone for guide me ?

**Comment by SomyaDhar** [comment]
Validation data did not given in kaggle .

But you can split validation data for train data set .
 
import this bellow code-
from sklearn.model_selection import train_test_split
and split by.........load all data in---- train_data
levels = y_train for you...............
x_train, x_valid, y_train, y_valid = train_test_split(train_data, labels, test_size = 0.1)

You can also give validation data size on ....
test_size =0.2

.......................................

however, you can also give validation data in the fit method......
like ----validation_split=0.1

h = model.fit(x=x_train, y=y_train, epochs=20,validation_split=0.1, batch_size=128)

**Comment by MahmodulRatul** [comment]
> Validation data did not given in kaggle .
> 
> But you can split validation data for train data set .
> 
> import this bellow code-
> from sklearn.model_selection import train_test_split
> and split by.........load all data in---- train_data
> levels = y_train for you...............
> x_train, x_valid, y_train, y_valid = train_test_split(train_data, labels, test_size = 0.1)
> 
> You can also give validation data size on ....
> test_size =0.2
> 
> .......................................
> 
> however, you can also give validation data in the fit method......
> like ----validation_split=0.1
> 
> h = model.fit(x=x_train, y=y_train, epochs=20,validation_split=0.1, batch_size=128)

Uncountable thanks to you brother
